camera_driver:
  ros__parameters:
    # Low-Level hardware access config
    libcamera_resolution_width: 1920
    libcamera_resolution_height: 1080
    libcamera_resolution_framerate: 30
    libcamera_verbose: true
    # Metadata
    frame_id: camera_sensor_link
    sample_period: 0.2
    # Publish Config
    publish_image_raw: True
    image_raw_topic: /camera/image
    publish_image_compressed: False
    image_compressed_topic: /camera/image/compressed

